.github/workflows/action-image-scan-deploy-dashing.yaml

```diff
@@ -0,0 +1,31 @@
+on: 
+  push:
+    branches:
+      - master
+      - main
+
+env:
+  ROS_VERSION: 2
+  UBUNTU_DISTRO: bionic
+  ROS_DISTRO: foxy
```

Comment:
Isn't it dashing?

Reply:
fixed on 8274b3cdc453a9a8844a8b335d39fa41efd8b20b